src/panels/Scene/SceneGraphNode/SceneGraphNodeView.tsx

```diff
@@ -99,6 +129,44 @@ export function SceneGraphNodeView({ uri }: Props) {
           )}
         </Tabs.Panel>
 
+        {/* @TODO (2025-03-19, emmbr): Add a way to display the different intervals that
+            the time frame, as human readable time stamps */}
+        {timeFrame && (
+          <Tabs.Panel value={TabKeys.TimeFrame}>
+            <Box p={'xs'}>
+              <Group gap={'xs'}>
+                <Text>Current status:</Text>
+                <Tooltip
+                  label={
+                    isInTimeFrame
+                      ? 'This object is currently active and will be visible'
+                      : 'This object is currently inactive due to its time frame and will not be visible'
+                  }
+                >
+                  <Badge
+                    size={'lg'}
+                    rightSection={
+                      isInTimeFrame ? (
+                        <ClockIcon size={IconSize.xs} />
+                      ) : (
+                        <ClockOffIcon size={IconSize.xs} />
+                      )
+                    }
+                    variant={'outline'}
+                    color={isInTimeFrame ? 'green' : 'red'}
+                  >
+                    {isInTimeFrame ? 'Active' : 'Inactive'}
+                  </Badge>
+                </Tooltip>
+              </Group>
+              <Text mt={'xs'} size={'sm'} c={'dimmed'}>
+                The object will not be visible outside the specified time range.
+              </Text>
+            </Box>
+            <PropertyOwner uri={timeFrame.uri} />
```

Comment:
I guess this is information coming from OpenSpace rather than frontend. But can we get which unit and timeframe (seconds since J2000 ?) in the tooltip for start and end input fields. Hard to know what the start and end times represent right now

Reply:
Thats an update that needs to be done in OpenSpace, so not related to this PR.

A discussed solution is to add a viewoption for numbers that represent time (which are currently always seconds since J2000). The time frame property owners is only ever showed for advanced users so I don't think it's very urgent to fix. 

It would be nice to also get a string representation of the dates where the time frame is active, but that's also something that we need to build in OpenSpace since there are different time frame types - not only a simple interval like the one used for JWST